src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -1322,15 +1331,62 @@ static void defaults(XmlWriter& xml, const MStyle& s, double& millimeters, const
+        xml.startElement("system-dividers");
+        if (s.styleB(Sid::dividerLeft)) {
+            xml.tag("left-divider", { { "print-object", "yes" },
+                        { "relative-x", s.styleD(Sid::dividerLeftX) * 10 },
+                        { "relative-y", s.styleD(Sid::dividerLeftY) * 10 } });
+        } else {
+            xml.tag("left-divider", { { "print-object", "no" } });
+        }
+        if (s.styleB(Sid::dividerRight)) {
+            xml.tag("right-divider", { { "print-object", "yes" },
+                        { "relative-x", s.styleD(Sid::dividerRightX) * 10 },
+                        { "relative-y", s.styleD(Sid::dividerRightY) * 10 } });
+        } else {
+            xml.tag("right-divider", { { "print-object", "no" } });
+        }
+        xml.endElement();
+        xml.endElement();
+    }
+
+    {
+        xml.startElement("appearance");
+        // line width values in tenth
+        xml.tag("line-width", { { "type", "light barline" } }, s.styleD(Sid::barWidth) * 10);
+        xml.tag("line-width", { { "type", "heavy barline" } }, s.styleD(Sid::endBarWidth) * 10);
+        xml.tag("line-width", { { "type", "beam" } }, s.styleD(Sid::beamWidth) * 10);
+        xml.tag("line-width", { { "type", "bracket" } }, s.styleD(Sid::bracketWidth) * 10);
+        xml.tag("line-width", { { "type", "dashes" } }, s.styleD(Sid::lyricsDashLineThickness) * 10);
+        xml.tag("line-width", { { "type", "enclosure" } }, s.styleD(Sid::staffTextFrameWidth) * 10);
+        xml.tag("line-width", { { "type", "ending" } }, s.styleD(Sid::voltaLineWidth) * 10);
+        xml.tag("line-width", { { "type", "extend" } }, s.styleD(Sid::lyricsLineThickness) * 10);
+        xml.tag("line-width", { { "type", "leger" } }, s.styleD(Sid::ledgerLineWidth) * 10);
+        xml.tag("line-width", { { "type", "pedal" } }, s.styleD(Sid::pedalLineWidth) * 10);
+        xml.tag("line-width", { { "type", "octave shift" } }, s.styleD(Sid::ottavaLineWidth) * 10);
+        xml.tag("line-width", { { "type", "slur middle" } }, s.styleD(Sid::SlurMidWidth) * 10);
+        xml.tag("line-width", { { "type", "slur tip" } }, s.styleD(Sid::SlurEndWidth) * 10);
+        xml.tag("line-width", { { "type", "staff" } }, s.styleD(Sid::staffLineWidth) * 10);
+        xml.tag("line-width", { { "type", "stem" } }, s.styleD(Sid::stemWidth) * 10);
+        xml.tag("line-width", { { "type", "tie middle" } }, s.styleD(Sid::SlurMidWidth) * 10);
+        xml.tag("line-width", { { "type", "tie tip" } }, s.styleD(Sid::SlurEndWidth) * 10);
+        xml.tag("line-width", { { "type", "tuplet bracket" } }, s.styleD(Sid::tupletBracketWidth) * 10);
+        xml.tag("line-width", { { "type", "wedge" } }, s.styleD(Sid::hairpinLineWidth) * 10);
+        // note size values in percent
+        xml.tag("note-size", { { "type", "cue" } }, s.styleD(Sid::smallNoteMag) * 100);
+        xml.tag("note-size", { { "type", "grace" } }, s.styleD(Sid::graceNoteMag) * 100);
```

Comment:
What about adding ```cpp
            // distance values in tenth
            xml.tag("distance type=\"beam\"", s->styleD(Sid::beamDistance) * 10);
            xml.tag("distance type=\"hyphen\"", s->styleP(Sid::lyricsDashPad) * 10); // Or lyricsDashMinDistance or lyricsDashMaxDistance?
´´´

Reply:
For the latter one I wasn't sure what the correct corresponding value would be. Maybe even `lyricsDashMaxDistance`? That's why I didn't add either.

Reply:
`beamDistance` though seems pretty clear

Reply:
Also this:
```cpp
xml.tag("note-size type=\"grace-cue\"", s->styleD(Sid::graceNoteMag) * s->styleD(Sid::smallNoteMag) * 100);
```

Reply:
Couldn't find a value for `beamDistance`. It is also not loaded from engraving defaults: 
[ "beamSpacing" handled separately](https://github.com/musescore/MuseScore/blob/64cebb064fe444cb90caad6c7236990655714a8a/src/engraving/internal/engravingfont.cpp#L437)

Reply:
`beamDistance` (which is not in the SMuFL spec) was removed in 4.0